chapters/02-theory.Rmd

```diff
@@ -0,0 +1,158 @@
+# Theory {#mathematicalTheory}
+
+## Exact Kernel Density Estimation
+
+An exact Kernel Density Estimation can be calculated as
+
+\begin{equation}
+\widehat{f}_h(x) = \frac{1}{nh} \sum_{k=1}^n K\Big(\frac{x-x_k}{h}\Big)
+(\#eq:kde)
+\end{equation}
+
+where $K(x)$ is called the kernel function, which defines the range and size of influence of a single data point over the estimation. Most typically a simple Gaussian distribution is used as kernel function.
+The wider the kernel function is, the farther  is the influence of a single data point, this is characterized by the bandwidth paramter $h$. 
+
+The computational complexity of the exact KDE above is $\mathcal{O}(nm)$ where $n$ is the number of sample points to estimate from and $m$ is the number of evaluation points (the points where you want to calculate the estimate).
+
+There exist several methods to combat this complexity. 
+
+## Simple and linear binning
+
+The most straightforward way to decrease runtime is by limiting the number of sample points. This can be done by a binning routine, where the values at a smaller number of regular grid points are estimated from the original large number of sample points.
```

Comment:
[3] runtime vs computational complexity is not quite the same! There is the factor parallelization ;)

Reply:
so maybe use again computational complexity here